admin/src/Membership/MembershipPeriodsInput.js

```
@@ -1,132 +1,118 @@
+        const deleteIds = deleteSpans.map((s) => s.id).join(",");
+        const timestamp = new Date().getTime().toString();
+        addSpans.forEach(
+            (s, i) =>
+                (s.creation_reason = (
+                    timestamp +
+                    i +
+                    " gui_edit:" +
+                    auth.getUsername() +
+                    " replacing:" +
+                    deleteIds
+                ).slice(0, 255)),
```

Biome (1.9.4)

[error] The assignment should not be in an expression.

The use of assignments in expressions is confusing.
Expressions are often considered as side-effect free.

(lint/suspicious/noAssignInExpressions)